src/language-handlebars/printer-glimmer.js

```diff
@@ -115,12 +115,19 @@ function print(path, options, print) {
       );
     }
     case "MustacheStatement": {
-      const shouldBreakOpeningMustache = isParentOfSomeType(path, [
+      const isParentOfSpecifiedTypes = isParentOfSomeType(path, [
         "AttrNode",
         "ConcatStatement",
-        "ElementNode",
       ]);
 
+      const isChildOfElementNodeAndDoesNotHaveParams =
```

Comment:
RE: (moving to a thread)

> @dcyriller how it should behave?

IMHO it should be the second, as you just did. Because there is no reason to print differently a helper and a component.